### PR TITLE
test new pipeline

### PR DESCRIPTION
**Pull request by silveyy**

### CODE REVIEW

File: tools/ExtensionDownloader.ps1 [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+    $actualHash =  Get-FileHash -Path $fileToCheck -Algorithm $algorithm;
+    
+    write-host "Algorithm:" $actualHash.Algorithm;
+    write-host "Path:" $actualHash.Path;
+    write-host "ExpectedHash: ["$expectedHash"] ActualHash ["$actualHash.Hash"].";
+
+    if ($actualHash.Hash -ne $expectedHash)
+    {
+        Write-Error $message;
```
Comment (lindsayShap) [inline tools/ExtensionDownloader.ps1:179]: Is Write-Error enough to make this fail here?  I think you need to throw don't you?

File: pipelines/secure-scan.yml [hunk 2]
```diff
@@ -22,4 +35,43 @@ steps:
       EnableServices: true
       TreatSignatureUpdateFailureAs: Standard
       TreatStaleSignatureAs: Warning
-    continueOnError: true
\ No newline at end of file
+    continueOnError: true
```
Comment (lindsayShap) [inline pipelines/secure-scan.yml:38]: should we continueOnError here?

File: tools/ExtensionDownloader.ps1 (hunk 1 above)
Comment (silveyy) [reply tools/ExtensionDownloader.ps1:179]: It actually is.... I had the same question so pushed this commit https://github.com/microsoft/configmgr-hub-selfhost/pull/623/commits/09f6b2a60524bac8414e1668fb8d14d738574d26 to check it and the pipeline failed. I'd give you a link to that pipeline if I could figure out how to match that run to that commit.  I figure you trust me though so I stopped looking for it.

File: pipelines/secure-scan.yml (hunk 2 above)
Comment (silveyy) [reply pipelines/secure-scan.yml:38]: Good question. Actually the PostAnalysis task below will check the logs and fail on error so I think this is correct.

**Review by lindsayShap — APPROVED** [review]

File: tools/ExtensionDownloader.ps1 (hunk 1 above)
Comment (silveyy) [reply tools/ExtensionDownloader.ps1:179]: Oh, and I figured out how to find the check that was performed that was linked to the commit where I was testing that.  Here it is

https://github.com/microsoft/configmgr-hub-selfhost/pull/623/checks?sha=09f6b2a60524bac8414e1668fb8d14d738574d26

**Pull request merged**